hudi-common/src/main/java/org/apache/hudi/common/config/HoodieStorageConfig.java

```diff
@@ -106,6 +106,15 @@ public class HoodieStorageConfig extends HoodieConfig {
           + "to be appended to a log file. This helps to make sure the data appended to the log file is broken up "
           + "into sizable blocks to prevent from OOM errors. This size should be greater than the JVM memory.");
 
+  public static final ConfigProperty<Boolean> LOGFILE_WRITE_RECORD_POSITIONS = ConfigProperty
```

Comment:
this is not a storage config? i.e its about a higher layer functionality around updates. Move this to where the core write configs are? and call it `WRITE_RECORD_POSITIONS`